pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,18 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// HandlerWithName holds information for handler and the Name of the plugin
+type HandlerWithName struct {
```

Comment:
why are these exported types now?

Reply:
operation_generator needs to instantiate the struct in registerPluginFunc